taxpose/nets/transformer_flow.py

```diff
@@ -541,12 +545,31 @@ def __init__(
                 nn.Conv1d(emb_dims_sym * 4, self.emb_dims, kernel_size=1, bias=False),
             )
 
-    def forward(self, *input):
-        action_points = input[0].permute(0, 2, 1)[:, :3]  # B,3,num_points
-        anchor_points = input[1].permute(0, 2, 1)[:, :3]
-
-        action_points_dmean = action_points - action_points.mean(dim=2, keepdim=True)
-        anchor_points_dmean = anchor_points - anchor_points.mean(dim=2, keepdim=True)
+    def forward(self, *input, conditioning_action=None, conditioning_anchor=None, action_center=None, anchor_center=None):
+        action_points = input[0].permute(0, 2, 1)[:, :self.input_dims]  # B,3,num_points
+        anchor_points = input[1].permute(0, 2, 1)[:, :self.input_dims]
+
+        if action_center is None:
```

Comment:
taxposed vae encoder will output distributions over the action/anchor point clouds which we sample a point from each to encode the demo. Jenny's found some benefit to centering the point clouds about the sampled points instead of mean, so on the taxposed side we pass in either the mean or sampled points for action/anchor point cloud centering as action_center/anchor_center